.github/workflows/build-spack.yml

```diff
@@ -239,7 +239,7 @@ jobs:
       - name: Regenerate ROOT test data
         if: >-
           ${{
-            matrix.geant == '11.0'
+            matrix.geant == '11.3'
```

Comment:
In a follow-up (since it will probalby require more tests updating) we need to change all the "11.0" in the `jobs/spack/strategy/matrix/include` to 11.3.